pandas/tests/indexing/test_timedelta.py

```diff
@@ -13,8 +13,7 @@ def test_boolean_indexing(self):
                          [0, 1, 2, 10, 4, 5, 6, 7, 8, 9],
                          [10, 10, 10, 3, 4, 5, 6, 7, 8, 9]]
         for cond, data in zip(conditions, expected_data):
-            result = df.copy()
-            result.loc[cond, 'x'] = 10
+            result = df.assign(x=df.mask(cond, 10).astype('int64'))
```

Comment:
@jreback do you remember what motivated this change?  it makes it harder to tell what is being tested